pkg/users/account-create-dialog.js

```diff
@@ -215,57 +294,74 @@
 
         errs.user_name = validate_username(user_name, accounts);
 
-        return password_quality(password, force)
-                .catch(ex => {
-                    errs.password = (ex.message || ex.toString()).replaceAll("\n", " ");
-                })
+        if (!state.home_dir)
+            errs.home_dir = _("No home directory specified");
+
+        const promises = [
+            password_quality(password, force_weak)
+                    .catch(ex => {
+                        errs.password = (ex.message || ex.toString()).replaceAll("\n", " ");
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18234-20230221-092948-26bd4da0-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)